### PR TITLE
workflow: rename mnemonic -> recovery words

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: src/workflow/restore_from_mnemonic.c [hunk 1]
```diff
@@ -143,11 +143,11 @@ bool workflow_restore_from_mnemonic(const RestoreFromMnemonicRequest* request)
     UTIL_CLEANUP_32(seed);
     size_t seed_len = 0;
     if (!keystore_bip39_mnemonic_to_seed(mnemonic, seed, &seed_len)) {
-        workflow_status_blocking("Invalid mnemonic", false);
+        workflow_status_blocking("Recovery words\ninvalid", false);
```
Comment (x1ddos) [inline src/workflow/restore_from_mnemonic.c:146]: I guess it doesn't fit on one line?

Comment (x1ddos) [reply src/workflow/restore_from_mnemonic.c:146]: Or how about "Invalid recovery words" to emphasize **invalid**?

Comment (benma) [reply src/workflow/restore_from_mnemonic.c:146]: yeah, doesn't fit on one line.

the status screen also has a big cross, so it is very emphasized :)

File: src/workflow/show_mnemonic.c [hunk 2]
```diff
@@ -33,8 +33,8 @@
 
 #define NUM_RANDOM_WORDS 5
 
-static const char* _back_label = "Back to seed phrase";
-static const char* _cancel_confirm_title = "Mnemonic";
+static const char* _back_label = "Back to\nrecovery words";
+static const char* _cancel_confirm_title = "Recovery\nwords";
```
Comment (x1ddos) [inline src/workflow/show_mnemonic.c:37]: This could still fit on one line though, diesn't it?

Comment (benma) [reply src/workflow/show_mnemonic.c:37]: No, it is a bit too close to the `x` and checkmark in the title.

**Pull request merged**